src/ucp/wireup/wireup.c

```diff
@@ -292,6 +292,35 @@ ucp_wireup_find_remote_p2p_addr(ucp_ep_h ep, ucp_lane_index_t remote_lane,
     return UCS_ERR_UNREACHABLE;
 }
 
+static ucp_lane_index_t
+ucp_wireup_ep_lane_used_by_another_ep_config(ucp_ep_config_key_t *ep_config_key,
```

Comment:
no need for this func, it is called just once from the function which does nothing else than calling it

Reply:
it will be used here:
https://github.com/openucx/ucx/blob/c65ba3b70a59d93775841506345a0076d1122b55/src/ucp/wireup/wireup.c#L553

Reply:
then why need this wrapper `ucp_wireup_ep_lane_used_by_another_ep`, if it just calls another func?

Reply:
removed

Reply:
`ucp_wireup_ep_lane_used_by_another_ep` is still present

Reply:
> `ucp_wireup_ep_lane_used_by_another_ep` is still present

sorry forgot to push the changes